meta-freescale-mbl/recipes-bsp/u-boot/u-boot-imx_%.bbappend

```diff
@@ -5,7 +5,11 @@
 inherit mbl-uboot-sign
 
 SRCBRANCH = "imx_v2018.03_4.14.78_1.0.0_ga-mbl"
-SRC_URI = "git://git.linaro.org/landing-teams/working/mbl/u-boot.git;protocol=https;nobranch=1 \
+SRC_URI = "git://source.codeaurora.org/external/imx/uboot-imx.git;protocol=https;nobranch=1 \
+    file://0001-imx8mm_evk_mbl-Add-an-mbed-Linux-imx8mm_evk-defconfi.patch \
```

Comment:
The u-boot-imx recipe is only used of imx8. Its now somewhat confusing that the tree contains u-boot imx8 patches in 2 places:
```
- /layers/meta-mbl/meta-freescale-mbl/recipes-bsp/u-boot/u-boot-imx
- /layers/meta-mbl/meta-freescale-mbl/recipes-bsp/u-boot/u-boot-imx/ imx8mmevk-mbl
```
this comment is for future reference as there isnt time to fix it now.